### DISCUSSION TITLE
Post Install script for co-installed PVE/PBS scenario

### DISCUSSION THREAD

**Discussion by kevinfason** [community-scripts/ProxmoxVE#338] [Ideas]
I think it would be valuable for the Post install PVE script to detect if PBS is co-installed and run those relevant PBS post-install changes as well. Or maybe a separate script for this scenario?

**Comment by Mellowlynx** [comment]
There is a separate PBS script that you can run within the PBS shell.
Does not matter if it's the LXC or bare metal.

Don't see why we would over complicate it.

**Comment by kevinfason** [comment]
I am referring to Installing Proxmox Backup Server on Proxmox VE. Same host, not installed separately via container or VM. Basically update repositories and apt install proxmox-backup-server on PVE host.

There are some things different between the post PVE and PBS scripts but also a lot of overlap.

**Reply by Mellowlynx** [reply]
As this is not the intentend wat to install and use PBS, we can't support it.

**Comment by kevinfason** [comment]
No worries. Appreciate the consideration. I got what was relevant from the two scripts but thought an aware option would be valuable to others.

To be devil's advocate, the recommended install is bare metal with local storage, not LXC nor VM that many of us homelabbers use. While Co-install is lower usage than LXC/VM use-cases, it is covered in the install directions.
https://pbs.proxmox.com/docs/installation.html#install-proxmox-backup-server-on-proxmox-ve

**Comment by LeonGrim2024** [comment]
I ran the Post Install script 30 minutes ago on my HP 800 G2. How long should the server take to start?
